web/html/src/manager/login/login.tsx

```diff
@@ -54,6 +54,23 @@ const getGlobalMessages = (validationErrors, schemaUpgradeRequired) => {
     messages = messages.concat({ severity: "error", text: schemaUpgradeError });
   }
 
+  if (diskspaceSeverity == "misconfiguration") {
+    const directoryMissing = t(
+      "Some important directories are missing. Please review your server configuration."
+    );
+    messages = messages.concat({ severity: "warning", text: directoryMissing });
+  } else if (diskspaceSeverity == "alert") {
+    const alertDiskSpace = t(
+      "The avalable disk space on the server is running low. Please add more space."
+    );
+    messages = messages.concat({ severity: "warning", text: alertDiskSpace });
+  } else if(diskspaceSeverity == "critical") {
+    const criticalDiskSpace = t(
+      "The available disk space on the server is critically low. Please add more space."
+    );
+    messages = messages.concat({ severity: "error", text: criticalDiskSpace });
+  }
```

Comment:
Something along those lines would be better:

```suggestion
  const severity_messages = {
    misconfiguration: Messages.warning(t("Some important directories are missing. Please review your server configuration.")),
    alert: Messages.warning(t("The avalable disk space on the server is running low. Please add more space.")),
    critical: Messages.error(t("The available disk space on the server is critically low. Please add more space.")),
  }
  messages = messages.concat(severity_messages[diskspaceSeverity]);
```

Reply:
Good point, I refactored the code in my last commit.